src/tableview.js

```diff
@@ -1,13 +1,16 @@
 import {NodeSelection} from 'prosemirror-state';
 import {addBottomRow, addRightColumn} from './commands';
-import {createButtonWithIcon, createElementWithClass} from './util';
+import {
+  createElementWithClass as cewc,
+  createElementWithClassAndChild as cewcac,
+} from './util';
 import {typeInheritance} from './headers/headers-menu/index';
 import {tableFiltersMenuKey} from './filters/utils';
 import {tableHeadersMenuKey} from './columnsTypes/types.config';
 
 const createAddCellsButton = (type, view, pos) => {
   const isRow = type === 'row';
-  const newElement = createElementWithClass(
+  const newElement = cewc(
```

Comment:
why for? very hard to read that

Reply:
I agree, we should definitely use the full names